server/tasks/process_file.py

```diff
@@ -29,6 +29,39 @@
 )
 
 
+def parent_job_error_handler(self, exc, task_id, args, kwargs, einfo):
+    """
+    This function will be called if streaming_parent_task fails, to ensure that we cleanup any left over
+    Kafka topics.
+
+    Also attempts to stop input/output jobs if their IDs are available
+    """
+    parent_job_id = task_id
+    input_topic_name = get_input_topic_name(parent_job_id)
+    output_topic_name = get_output_topic_name(parent_job_id)
+    delete_topic(input_topic_name)
+    delete_topic(output_topic_name)
+
+    parent_job = streaming_parent_task.AsyncResult(parent_job_id)
+    if (
+        parent_job.info is None
+        or type(parent_job.info)
+        != dict  # In some failure cases e.g. an exception is thrown, job.info will be a string, causing the next line to crash
+        or "input_job_id" not in parent_job.info
```

Comment:
Can there be a case where we only have the input job, or only have the output job? If so, I figure we should still try to delete the one that we have